main.py

```diff
@@ -0,0 +1,70 @@
+"""
+You could download multiple courses at once 
+with help of the script from stepic.org .
+
+
+Run by: 
+python3 main.py
+
+Additional notes:
+1) Add courses which you want to download to the "courses" dictionary.
+
+2) You could also set quality of the video by changing "quality" variable. 
+The default quality is 720.
+"""
+
+import os
+
+import weekDownloader
+
+def main():
+    # Set the quality of a video.
+    quality = '720'
+
+    courses = {}
+
+    # Add courses.
+    # courses[course_id] = [week_1, week_2, ...]
+
+    # Course example.
+    courses["154"] = ["1", "2", "3"]
```

Comment:
1) I will fix it by reverting back to the old version.
2) I will fix it by moving _quality_ and _courses_ to the setting file.

Then delete the old main function.

In the end we will have only one option to set data - _settings file_.